indico/modules/events/surveys/fields/base.py

```diff
@@ -67,10 +69,16 @@ def save_config(self, form):
                                     for name, field in form._fields.iteritems()
                                     if name not in form._common_fields and name != 'csrf_token'}
 
-    def get_wtforms_field(self):
+    @property
+    def wtf_field(self):
```

Comment:
to be honest, I'm not a big fan of using a property for this. It *creates* a new field, and while doing so is pretty cheap that's not the behavior I'd expect from a property. For example, `foo.wtf_field is foo.wtf_field` would most likely return `False`